app/js/components/ReactReduxSelect.jsx

```diff
@@ -0,0 +1,36 @@
+// ----------------------------------------------------------------------------
+// Copyright (c) 2016-2017, QIIME 2 development team.
+//
+// Distributed under the terms of the Modified BSD License.
+//
+// The full license is in the file LICENSE, distributed with this software.
+// ----------------------------------------------------------------------------
+
+import React from 'react';
+import Select from 'react-select-plus';
+import style from '!style-loader!css-loader!react-select-plus/dist/react-select-plus.css'; // eslint-disable-line
+
+
+class ReactReduxSelect extends React.Component {
```

Comment:
Maybe a different name would be better, since `Redux` doesn't really come into it.